shipit.yml

```diff
@@ -12,7 +14,8 @@ ci:
 
 merge:
   require:
-    - 'Travis CI - Branch'
+    - 'Test, lint and deploy'
+    - 'Accessibility Tests'
```

Comment:
I feel like we don't need visual regression for this step in shipit. Should it be added to ignore?